CHANGELOG.md

```diff
@@ -28,12 +28,13 @@
 - Support PodDisruptionBudgets ([#330]).
 - Added support for versions 2.6.3, 2.7.2 ([#334]).
 - Support graceful shutdown ([#343]).
+- Use new label builders ([#364]).
```

Comment:
Please move this to the `[Unreleased]` section ;)
(same for other entries and link)

Reply:
fixed in 2097e68eadd31f959018d7d2cac2d4fce8734c38